das/worker.go

```diff
@@ -60,8 +60,16 @@ func (w *worker) run(
 		}
 
 		metrics.observeGetHeader(ctx, time.Since(startGet))
-		log.Debugw("got header from header store", "height", h.Height(), "hash", h.Hash(),
-			"square width", len(h.DAH.RowsRoots), "data root", h.DAH.Hash(), "finished (s)", time.Since(startGet))
+
+		// TODO(@team): find linter rule to enforce this
```

Comment:
there is a linter rule to enforce that a LOC doesn't exceed certain char amount, so you can remove this todo

Reply:
I meant more like, the style of having `"attribute", var` per line than the LOC length. Seems that such rule doesn't exist.